src/decorator/decorators.ts

```diff
@@ -53,6 +53,7 @@ export function Validate(constraintClass: Function, constraintsOrValidationOptio
  */
 export function ValidateNested(validationOptions?: ValidationOptions) {
     return function (object: Object, propertyName: string) {
+        IsDefined()(object, propertyName);
```

Comment:
mmm maybe its better to left use to decide if this nested property should be defined or not? So, if `@ValidateNested()` and property is missing then nothing happens. If `@ValidateNested()` + `@IsDefined()` is used on property then if its missing it will throw an error. What do you think about it?